spec/get_visits_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'get all visits route', type: :request do
+  let!(:visits) { FactoryBot.create_list(:random_visit, 20) }
+
+  before { get '/api/v1/visits' }
+
+  it 'returns all visits' do
```

Comment:
As we decided, we remove controller tests at this moment. We will back to it later